README.md

```diff
@@ -1 +1,74 @@
-A simple script to genrate rpms from sddm theme tar files from pling.
\ No newline at end of file
+A simple script to generate RPMs from sddm theme tar files from pling.
+
+## Importance
+Fedora Distributions using OSTree have a read-only system image that is downloaded and can only be modified via `rpm-ostree override` on your device (without actually hosting your own repository).
+
+Currently this command can only remove or add RPM packages, not change, remove or add system files manually, which limits the customizability of these Distributions a lot, even though OSTree is a perfect solution for a solid system.
+
+What this tool does is creating an RPM whose only content is the SDDM theme, placed at the correct location. 
+
+## Usage in Fedora OSTree (Silverblue, Kinoite,...)
```

Comment:
Add instructions for MicroOS

Reply:
also I have uploaded it to https://crates.io https://crates.io/crates/sddm2rpm which should make installation simpler